misc/README.md

```diff
@@ -0,0 +1 @@
+This folder contains tests which depend on external modules.
```

Comment:
Maybe a bit clearer, based on PR description:

```suggestion
# misc

This folder contains tests which depend on external modules.
These are in a separate module to avoid polluting the main module dependency graph.
```

(I feel like it would be nice if the folder name were more descriptive, but I see there's precedent in the main Go repo for misc.)